### ISSUE TITLE
Implement a functionality so that users can provide the climatological values for GAMMA in the steps blending

### ISSUE THREAD

**Issue by sidekock** [pySTEPS/pysteps#448]
In the folliwing part of the code something should be added/changed:
Get the climatological lag-1 and lag-2 auto-correlation values from Table 2
in `BPS2004`.
Hard coded, change to own (climatological) values when present.
"#" TODO: add user warning here so users can be aware of this without reading the code?

This todo summarized the current problem. First suggestion by @RubenImhoff is to:
Should we build in a functionality so that users can provide the climatological values for GAMMA, e.g. through the kwargs?

Recomended to only start working on this once the refactor-steps-blending is merged into the master
